hw-5/server.ts

```diff
@@ -7,14 +7,29 @@ import userRouter from './routers/user.router';
 
 import express = require('express');
 import bodyParser = require('body-parser');
+import {apiLogger} from './middlewares/api-logger';
+import {errorHandler} from './middlewares/error-handler';
+import {logger} from './middlewares/logger';
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 app.use(bodyParser.json());
+app.use(apiLogger);
 app.use('/users', userRouter);
 app.use('/groups', groupRouter);
+app.use(errorHandler);
 
 app.listen(PORT, () => {
-  console.log(`Server is listening on http://localhost:${PORT}`);
+  logger.info(`Server is listening on http://localhost:${PORT}`);
 });
+
+process.on('unhandledRejection', function(reason) {
+  logger.error('unhandledRejection', reason);
+});
+
+process.on('uncaughtException', function(err) {
+  logger.error('UncaughtException:', err.message);
```

Comment:
I believe the `:` character is not needed.

Reply:
Right! Thanks!